README.md

```diff
@@ -0,0 +1,59 @@
+# Operator Framework for Charms
+
+This framework is not yet stable and is subject to change, but is available
+for early testing.
+
+## Getting Started
+
+Start by creating a charm directory with at least the following files:
+
+* `lib/charm.py` (must be executable and use Python 3.6+)
+* `hooks/install` (or `hooks/start` for K8s charms) sym-linked to `../lib/charm.py`
+* `metadata.yaml`
+
+Then install the framework into the `lib/` directory using:
```

Comment:
We should mention that standard Python idioms, such as using a virtualenv won't work because we need to transfer the charm to a remote host for execution.